apps/src/lib/client/rpc.rs

```diff
@@ -1892,23 +1882,26 @@ pub async fn query_conversions(ctx: Context, args: args::QueryConversions) {
 }
 
 /// Query a conversion.
-pub async fn query_conversion(
-    client: HttpClient,
+pub async fn query_conversion<CLIENT>(
```

Comment:
I needed these changes for benchmarks since I wanted to reuse some client functions of masp (specifically to construct shielded txs) but without a real client nor a running ledger. I suspect the SDK guys are doing something similar to this but I haven't has the time to check

Reply:
yeah, it's exactly like that